src/org/omegat/core/team2/impl/SVNAuthenticationManager.java

```diff
@@ -232,4 +239,46 @@ public String getProxyUserName() {
         public void acknowledgeProxyContext(boolean accepted, SVNErrorMessage errorMessage) {
         }
     };
+
+    private SVNAuthentication getResult(String kind, SVNURL url, String user, char[] pass)
+            throws SVNException {
+        if (ISVNAuthenticationManager.PASSWORD.equals(kind)) {
+            return SVNPasswordAuthentication.newInstance(user, pass, false, url, false);
+        } else if (ISVNAuthenticationManager.SSH.equals(kind)) {
+            return SVNSSHAuthentication.newInstance(user, pass, -1, false, url, false);
+        } else if (ISVNAuthenticationManager.USERNAME.equals(kind)) {
+            return SVNUserNameAuthentication.newInstance(user, false, url, false);
+        } else {
+            throw new SVNException(SVNErrorMessage.create(SVNErrorCode.RA_UNKNOWN_AUTH));
+        }
+    }
+
+    private Credentials loadCredentials(SVNURL url) {
+        Credentials credentials = new Credentials();
+        // check stored credential
+        credentials.username = TeamSettings.get(repoDef.getUrl() + "!" + KEY_USERNAME_SUFFIX);
```

Comment:
We use team setting here but we drop teamSetting argument in SVNAuthenticationManager constructor.
It would be better to keep ProjectTeamSettings and checks it.

Reply:
projectTeamSettings don't have a credential record.